loadtest.py

```diff
@@ -58,25 +60,27 @@ def make_example_shot():
                 ),
             ),
         },
-)
+    )
 
 text_strings = """
 Example strings like apple orange banana some stuff like whatever and whoever and bucket blanket funky etc keyboard screen house window tree leaf leaves feather feathers
 """.split()
 
+
 @scenario(100)
 async def create_shot(session):
 
-    #async with session.put(SERVER_URL + STATUS_URL) as r:
-    #    body = await r.json()
-    #    assert 'user' in body
+    # async with session.put(SERVER_URL + STATUS_URL) as r:
+    #     body = await r.json()
+    #     assert 'user' in body
 
     shot_id = make_uuid() + "/test.com"
     shot_url = urljoin(SERVER_URL, shot_id)
     shot_data = urljoin(SERVER_URL, "data/" + shot_id)
     resp = make_example_shot()
-    async with session.put(shot_data, data=resp) as r:
+
+    async with session.put(shot_data, json=resp) as r:
```

Comment:
http://docs.python-requests.org/en/master/user/quickstart/#more-complicated-post-requests:

> Instead of encoding the `dict` yourself, you can also pass it directly using the `json` parameter (added in version 2.4.2) and it will be encoded automatically:
>
> ```
> >>> url = 'https://api.github.com/some/endpoint'
> >>> payload = {'some': 'data'}
> 
> >>> r = requests.post(url, json=payload)
> ```